src/main/java/robot/Constants.java

```diff
@@ -61,6 +59,17 @@ public static void replaceFields(Class class1, Class class2) {
         }
     }
 
+    public class Wrist {
+        public static final int TIMEOUT_MS = 20;
+        public static final int PID_SLOT = 0;
+        public static final double KP = 0;
+        public static final double KI = 0;
+        public static final double KD = 0;
```

Comment:
Can be written as a single array of doubles
`        public static final double[] CLIMB_PIDF = {0, 0, 0, 0};
`